reactioncommerce/reaction
# Fixed missing placeholders on Social textfields in Product Detail Settings

Resolves #3016 ,

The textfield components for Social sharing were missing the `i18nKeyPlaceholder` property in their components, this also caused the default text for share messages to be `undefined`.

#### How to test placeholder fix
- Login as an admin 
- Select a product.
- Click on the edit social settings icon as seen in the image below with the blue square highlighting the icon,
<img width="928" alt="placeholder-fix-screenshot" src="https://user-images.githubusercontent.com/28437833/31440921-706f0c20-ae89-11e7-8936-ce895f9fb19a.png">
- As seen in the image above all the social textfields have their placeholders displaying

#### How to test default text also displays
We will use Twitter for this test
- Login as an admin. 
- Select a product.
- Click on the twitter icon to share as shown in the image below with the oval circle highlighting it,
<img width="985" alt="share-with-twiiter" src="https://user-images.githubusercontent.com/28437833/31441187-53e0299e-ae8a-11e7-8f92-63891694b18b.png">
- Notice in the image below that a default text `ie. Injustice` is in the twitter share message as seen below,
<img width="993" alt="twiitter-example-screenshot" src="https://user-images.githubusercontent.com/28437833/31441034-dc93b3c4-ae89-11e7-914a-3c51aabe7b5d.png">
